src/UserClassifier03.py

```diff
@@ -1,8 +1,5 @@
 """
-perspektivisch: 1) pro kategorie ein Track/Synthsizer?
-2) bei neuen Texten: a) in UserOrdner boris/Zustimmung.txt b) alle_user/Zustimmung.txt
-vor allem die b) lösung wäre eine wachsende Datenbank, die zusätzlich zu meiner bisherigen TrainingData
-(die ja letztlich eine Boris-Datenbank ist) hinzugezogen wird.
+
```

Comment:
This now empty comment block can be removed, no?

Reply:
yup